exporters/etw/src/etw_tracer_exporter.cc

```diff
@@ -0,0 +1,7 @@
+#ifdef _WIN32
+
+#  define HAVE_NO_TLD
```

Comment:
Should this be defined here, when we also have it defined in a `CMakeLists.txt` file? Otherwise this will always override what comes from CMake.

Reply:
This is kept for now to keep Bazel build happy. This will be removed once we OSS the `TraceLoggingDynamic.h` header. Equivalent TLD functionality is available in Go, for example, here: https://godoc.org/github.com/microsoft/go-winio/pkg/etw .. so we will democratize the proper full support of ETW events and this define will go away.